docs/end-user/components/ref-objects.md

```diff
@@ -1,5 +1,103 @@
 ---
-title: Distributed External Kubernetes Objects
+title: Distribute Reference Objects
```

Comment:
Distribute the kubernetes manifests to other clusters

Reply:
this is a title